scripts/pin_the_slow_way.py

```diff
@@ -29,22 +29,22 @@
 
 
 pinned = {
-          'boost': 'boost 1.60.*',
-          'bzip2': 'bzip2 1.0.*',
-          'fontconfig': 'fontconfig 2.11.*',
+          'boost': 'boost 1.61*',
```

Comment:
Yeah, that is how I do my pinning too, but I have seen this new form more (in the wild) than I have seen my preferred form. I have never got around to it, but I wonder whether `1.60` gets installed if I specify `1.6*`?